docs/src/components/GettingStartedSearch.tsx

```diff
@@ -11,6 +11,7 @@ const keywordColors = {
     'inline-flex items-center rounded-md bg-yellow-50 dark:bg-yellow-400/10 px-2 py-1 text-xs font-medium text-yellow-600 dark:text-yellow-500 ring-1 ring-inset ring-yellow-500/10 dark:ring-yellow-400/20',
   rust: 'inline-flex items-center rounded-md bg-orange-50 dark:bg-orange-400/10 px-2 py-1 text-xs font-medium text-orange-600 dark:text-orange-500 ring-1 ring-inset ring-orange-500/10 dark:ring-orange-400/20',
   go: 'inline-flex items-center rounded-md bg-cyan-50 dark:bg-cyan-400/10 px-2 py-1 text-xs font-medium text-cyan-600 dark:text-cyan-500 ring-1 ring-inset ring-cyan-500/10 dark:ring-cyan-400/20',
+  java: 'inline-flex items-center rounded-md bg-amber-50 dark:bg-amber-400/10 px-2 py-1 text-xs font-medium text-amber-600 dark:text-amber-500 ring-1 ring-inset ring-amber-500/10 dark:ring-amber-400/20',
```

Comment:
nit: what about pink or something else for java, the amber looks really close to yellow for javascript